src/lib/seam/components/AccessCodeTable/AccessCodeTable.tsx

```diff
@@ -268,6 +264,20 @@ export function AccessCodeTable({
             disableDeleteAccessCode={disableDeleteAccessCode}
           />
         </TableBody>
+
+        <Snackbar
+          variant='error'
+          visible={isError}
+          onClose={() => {}}
```

Comment:
Looks like onClose could be an optional prop

Reply:
#526